test/ui-test/testSuites/suite_status/shared/scripts/sections/chat_names.py

```diff
@@ -2,6 +2,7 @@
 from sections.global_names import *
 
 # Main Window - chat related:
+mainWindow_chat_navbar = {"container": statusDesktop_mainWindow, "objectName": "Chat-navbar", "type": "StatusNavBarTabButton"}
```

Comment:
We use cammelCase for objectNames usually. It's not a big deal, it's just a bit weird to start mixing.

Reply:
oh okay

Reply:
Oh I forgot - its a complex objectName which someone add (`text + "-navbar"`)